shared/cfg/machines.cfg

```diff
@@ -46,3 +46,15 @@ variants:
         # Currently no USB support
         usbs =
         usb_devices =
+    - s390:
+        only s390
```

Comment:
PS: From avocado point of view the `machine-type` is the guest hardware platform and `arch` is the guest architecture. For example powerpc supports various machine types, but `pseries` is the most common. And this HW can run either little or big (`ppc64` or `ppc64le`) endian guest OS.